src/gss_names.c

```diff
@@ -281,6 +281,80 @@ uint32_t gssntlm_import_name(uint32_t *minor_status,
                                        output_name);
 }
 
+size_t gssntlm_get_attrs_count(const struct gssntlm_name_attribute *attrs)
+{
+    size_t attrs_count = 0;
+    if (!attrs) return 0;
```

Comment:
Here an in many other places below, the style guide says:
`We prefer if-statements to always use braces`
See: https://github.com/gssapi/gss-ntlmssp/wiki/Contributions#indentation

Reply:
Just to be clear, although the guide gives an escape hatch for the case 'if (err) goto done;' I think that is almost always the only exception, because it generally sits as the last statement of a section followed by a blank line.

Single line if statments are not a great read in the thick of a code section.

Reply:
ok